aiida/cmdline/commands/cmd_calcjob.py

```diff
@@ -41,7 +41,7 @@ def calcjob_gotocomputer(calcjob):
     try:
         transport = calcjob.get_transport()
     except NotExistent as exception:
-        echo.echo_critical(exception)
+        echo.echo_critical(repr(exception))
```

Comment:
Why `repr` and not `str`?

Reply:
It just includes information about the error as well:
```
In [1]: try:
   ...:     raise ValueError("abc")
   ...: except ValueError as er:
   ...:     print(str(er))
   ...:
abc

In [2]: try:
   ...:     raise ValueError("abc")
   ...: except ValueError as er:
   ...:     print(repr(er))
   ...:
   ...:
ValueError('abc')
```

Reply:
Ok - I'm not sure this is the best way to achieve this (I would have printed `str` and the type of the exception) but I think it serves the purpose here, for now